ui/raidboss/popup-text.js

```diff
@@ -227,8 +244,22 @@ class PopupText {
   }
 
   OnJobChange(e) {
-    this.me = e.detail.name;
-    this.job = e.detail.job;
+    // allow override of player via query parameter
+    // only apply override if player is in party
+    if (Options.PlayerNameOverride !== null &&
```

Comment:
I tested this out locally, and any time `OnPlayerChange` is called, then `data.me` no longer equals `e.detail.name` because it's set here, and so it reloads everything on every update.  I think it might make more sense to apply the override in `OnPlayerChange` and edit the event that gets passed to `OnJobChange`?

Reply:
Agreed. I'll see about making this adjustment tomorrow if I get time before raid.

Reply:
Moved this logic up to `OnPlayerChange` in latest changes.